Makefile

```diff
@@ -40,7 +40,7 @@ COVERAGE_PATH := $(BUILD_PATH)/coverage
 PLUGIN = $(BIN_PATH)/mixedcpu
 
 COMMONENVVAR = GOOS=linux GOARCH=amd64
-BUILDENVVAR = CGO_ENABLED=1
+BUILDENVVAR = CGO_ENABLED=0
```

Comment:
ok, but why?

Reply:
I should add the GH reference will do that asap